website/docs/docs/develop/getting-started/getting-started.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Getting started with dbt Cloud
 id: getting-started
```

Comment:
```suggestion
id: getting-started-dbt-cloud
```

Reply:
to align with `id: getting-started-dbt-core` for "Getting started with dbt Core"